data/pirate jobs.txt

```diff
@@ -2465,6 +2465,58 @@ mission "Cargo Theft (South)"
+				launch
+		government "Merchant"
+		personality staying uninterested timid target marked
+		system
+			distance 2 3
+		fleet
+			names "civilian"
+			variant
+				"Modified Argosy (Smuggler)"
+	on fail
+		dialog "You have failed to capture the smuggler and recover the stolen cargo. This will probably not be good for your reputation."
+		"reputation: Pirate" <?= -10
+	on offer
+		conversation
+			`As you step into a dive bar for a quick drink, the largest man you've ever seen stands up and blocks your path. "<first> <last>," he says in a thick accent. "My employer hears you do certain jobs, take money, do not ask questions. He hears you are reliable. A man, a smuggler, he steals cargo from my employer. My employer very much wants cargo returned, and this man as well."`
+			choice
+				`	"I'm sorry, sir, but I'm not taking new jobs today. Please give your employer my most gracious thanks for this kind offer."`
+				`	"Of course. I won't say a word."`
+					goto accept
+				`	"Get out of my way. I don't take jobs from any schlub who bothers me when I need a drink."`
+					goto mistake
+
+			`	The man looks down at you with no change in expression. "My employer, he will not be happy. I will tell him, '<first> <last> is not so reliable.' I hope next time, you are reliable." As the man speaks, you notice that his teeth are filed to vicious points. Every patron in the bar stares at you in shock as you sit down for a badly-needed drink.`
+				decline
+
+			label accept
+			`	The man looks down at you with no change in expression. "This is good. To be reliable, it is a great thing on <origin>." As the man speaks, you notice that his teeth are filed to vicious points. He hands you a datachip containing details of the job. As the man leaves, you see every patron in the bar staring warily at his back.`
+				accept
+
+			label mistake
+			`	The man looks down at you with no change in expression. "This is great shame. I will tell my employer, '<first> <last> was not so reliable. I had to make example.'" The man grabs your neck with one monstrous hand. The last thing you see as your vision fades to black is a mouth full of pointed teeth.`
+				die
+	on visit
+		dialog "You land on <origin>, but you don't have the smuggler and the cargo he stole. You should probably be careful not to run into your contact until you do."
+	on complete
+		payment 150000
+		conversation
+			`The huge man drags the unconscious smuggler out of the cargo container with one hand and carries the cargo box with the other. He stops in front of you. "<first> <last>," he says. "My employer, he will be very happy. Your payment, you will receive soon. Is it not good, to be reliable?" With one more shark-toothed grin, the man leaves your ship. A moment later, you receive a message: '<payment> have been transferred to you from an anonymous account.'`
```

Comment:
Perhaps change this last bit to `you receive <payment> from an anonymous account.`